lib/vagrant-parallels/action/export.rb

```diff
@@ -61,7 +61,8 @@ def clone(env)
           env[:ui].info I18n.t('vagrant.actions.vm.export.exporting')
 
           options = {
-            dst: env['export.temp_dir'].to_s
+            dst: env['export.temp_dir'].to_s,
+            linked: true,
```

Comment:
@sony-aufsehe You are right, the additional unregister is causing issue. But to resolve this, we can remove the unregister call from `copy_clone_vm `. Unregister will be done by the `BoxUnregister` action after `Import`.